composer.json

```diff
@@ -36,6 +36,7 @@
         "symfony/proxy-manager-bridge": "5.3.*",
         "symfony/runtime": "5.3.*",
         "symfony/security-bundle": "5.3.*",
+        "symfony/security-csrf": "5.3.*",
```

Comment:
Security-csrf is included in security-bundle so you probably don't need to add it